runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/AnagramClue.java

```diff
@@ -383,12 +429,14 @@ public class AnagramClue extends ClueScroll implements NpcClueScroll, ObjectClue
 			.answer("2")
 			.build(),
 		AnagramClue.builder()
+			.itemId(ItemID.CLUE_SCROLL_MASTER)
```

Comment:
Same feedback as emote clues, it seems odd to me to associate clues with real item IDs if we intentionally don't want to match them.